### DISCUSSION TITLE
Reset after animation operation

### DISCUSSION THREAD

**Discussion by hcxmj** [pmndrs/drei#1173] [Q&A]
After I use OrbitControls to achieve animation, I want to reset and set the regress effect to redraw. There will be a white screen interval in the middle. I hope to directly adjust the camera to achieve
Is it possible to configure directly through the API?
<img width="955" alt="image" src="https://user-images.githubusercontent.com/58494807/206116316-6aa8b751-a3be-41d9-bca4-89f35ff22099.png">

**Comment by drcmda** [comment]
i think this has too little information. white interval? what's onChange?

**Reply by hcxmj** [reply]
Just like the rendering interruption shown in the screen recording, the effect is similar to refreshing the page
[https://static-d.iqiyi.com/lequ/20221207/74a5299c056e460f80919973de8f6eb9.mov](url)
Video resources need to be copied to the browser to open

**Reply by drcmda** [reply]
the video doesnt load, i get a 404

**Reply by hcxmj** [reply]
ok, let me debug again,thank you @drcmda

**Reply by hcxmj** [reply]
Hi, @drcmda excuse me again;If I want to add a button to reset the camera position according to the above code, what should I do? I tried some methods and found that it didn’t work.

**Reply by hcxmj** [reply]
> the video doesnt load, i get a 404
Here is my code

`import React, { Suspense, useRef, useState } from 'react'
import { Canvas, useFrame, useThree } from '@react-three/fiber'
import { OrbitControls, Stage } from '@react-three/drei'

export default function Viewer({ contactShadow, autoRotate, environment, preset, intensity, scene,} : any) {
  const ref:any = useRef();

  return (
    <>
    <Canvas style={{ width: '100%', height: '100%' }} gl={{ preserveDrawingBuffer: true }}>
      <ambientLight />
      <Suspense fallback={null}>
        <Stage
          // @ts-ignore
          controls={ref}
          preset={preset}
          intensity={intensity}
          contactShadow={contactShadow}
          adjustCamera
          environment={environment}
        >
          <primitive object={scene} />
        </Stage>
      </Suspense>
      <OrbitControls
        // @ts-ignore
        ref={ref} 
        autoRotate={autoRotate}
      />
    </Canvas>
    </>
  )
}`
